### PR TITLE
chore: Removes documentation from README and link to docs on Charmhub instead

### PR DESCRIPTION
**Pull request by saltiyazan**
# Description

Improves the README, metadata and config files

Note for reviewers:
Please take a look at the enhanced docs in Charmhub:
- https://discourse.charmhub.io/t/route53-lego-operator-k8s-docs-index/12514

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [x] I validated that new and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,59 +1,12 @@
 # Route53 LEGO Operator (K8s)
 [![CharmHub Badge](https://charmhub.io/route53-lego-k8s/badge.svg)](https://charmhub.io/route53-lego-k8s)
 
-Let's Encrypt certificates in the Juju ecosystem for AWS route53 users.
+LEGO operator implementing the provider side of the 
+[`tls-certificates` interface](https://github.com/canonical/tls-certificates-interface) 
+to get signed certificates from the `Let's Encrypt` ACME server
+using [AMAZON ROUTE 53](https://go-acme.github.io/lego/dns/route53/) plugin and the DNS-01 challenge.
```
Comment (ghislainbourgeois) [inline README.md:7]:
```suggestion
using [Amazon Route 53](https://go-acme.github.io/lego/dns/route53/) plugin and the DNS-01 challenge.
```

File: metadata.yaml [hunk 2]
```diff
@@ -7,10 +7,12 @@ display-name: Route53 LEGO (K8s)
 
 description: |
   LEGO operator implementing the provider side of the `tls-certificates`
-  interface to get signed certificates from the `Let's Encrypt` ACME server using route53 DNS.
+  interface to get signed certificates from the `Let's Encrypt` ACME server
+  using AMAZON ROUTE 53 plugin and the DNS-01 challenge.
```
Comment (ghislainbourgeois) [inline metadata.yaml:11]:
```suggestion
  using Amazon Route 53 plugin and the DNS-01 challenge.
```

File: metadata.yaml [hunk 3]
```diff
@@ -7,10 +7,12 @@ display-name: Route53 LEGO (K8s)
...
 summary: |
   LEGO operator implementing the provider side of the `tls-certificates`
-  interface to get signed certificates from the `Let's Encrypt` ACME server using route53 DNS.
+  interface to get signed certificates from the `Let's Encrypt` ACME server
+  using AMAZON ROUTE 53 plugin and the DNS-01 challenge.
```
Comment (ghislainbourgeois) [inline metadata.yaml:15]:
```suggestion
  using Amazon Route 53 plugin and the DNS-01 challenge.
```

**Review by ghislainbourgeois — CHANGES_REQUESTED** [review]
Looks good, but I would not use all caps for the name of the service. Amazon uses the capitalization I suggested.

**Comment by gruyaume** [comment]
> Looks good, but I would not use all caps for the name of the service. Amazon uses the capitalization I suggested.

Unless you want to yell it loudly...

**Review by Kayra1 — APPROVED** [review]

**Review by ghislainbourgeois — APPROVED** [review]

**Pull request merged**